packages/synapse-react-client/test/lib/containers/entity_finder/EntityFinder.test.tsx

```diff
@@ -89,7 +89,6 @@ const defaultProps: EntityFinderProps = {
   initialContainer: 'syn123',
   selectMultiple: true,
   onSelectedChange: mockOnSelectionChange,
-  showVersionSelection: true,
```

Comment:
`showVersionSelection` is not currently an EntityFinder prop. Happy to add this back though, if it's included by design.